<launch>
  <node name="jog_frame_node" pkg="jog_controller" type="jog_frame_node">
    <param name="controller_name" value="position_trajectory_controller"/>
    <param name="target_link" value="link_j6"/>
    <param name="group_name" value="manipulator"/>
  </node>
  <node if="true" name="joy" pkg="joy" type="joy_node">
    <param name="autorepeat_rate" value="100"/>
    <param name="deadzone" value="0.1"/>
  </node>
  <node if="true" name="joy_to_jog_frame" pkg="jog_controller" type="joy_to_jog_frame.py">
    <param name="group_name" value="manipulator"/>
  </node>
</launch>
